public/app/plugins/datasource/testdata/datasource.ts

```diff
@@ -80,30 +80,16 @@ export class TestDataDataSource extends DataSourceApi<TestDataQuery> {
           streams.push(this.nodesQuery(target, options));
           break;
         default:
-          queries.push({
-            ...target,
-            intervalMs: options.intervalMs,
-            maxDataPoints: options.maxDataPoints,
-            datasourceId: this.id,
-            alias: this.templateSrv.replace(target.alias || '', options.scopedVars),
-          });
+          backendQueries.push(target);
       }
     }
 
-    if (queries.length) {
-      const stream = getBackendSrv()
-        .fetch({
-          method: 'POST',
-          url: '/api/tsdb/query',
-          data: {
-            from: options.range.from.valueOf().toString(),
-            to: options.range.to.valueOf().toString(),
-            queries: queries,
-          },
-        })
-        .pipe(map((res) => this.processQueryResult(queries, res)));
```

Comment:
Can we also remove the `processQueryResult` function and the unused `TestData` type?